test/type/implicit_some/opaque_parameters.swift

```diff
@@ -0,0 +1,108 @@
+// RUN: %target-typecheck-verify-swift -disable-availability-checking -warn-redundant-requirements  -enable-experimental-implicit-some
```

Comment:
You should test type aliases where the underlying type is a protocol type or composition, as well